docs/customizing.rst

```diff
@@ -0,0 +1,198 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Customizing
+===========
+
+
+how to inject/override the Invenio config (url, facets, the DOM id where to render the app): via parametrize or remove the Jinja macro and inject JSON config
```

Comment:
minor: due to the structure of the sentence it is a bit hard to grasp:

```suggestion
There are two options to customize the configuration of the search pages (URL, facets, DOM ids, etc.). Parametrizing the [...] or removing the jinja macro and injecting the JSON config into the bare element [...]
```

Basically, I read that and I honestly did not understand what was "parameterize" or the "JSON config" (of what?) a few code examples might shed some light (most likely references to the content below is enough).

Reply:
Ah, well this was a leftover from the original issue. I was intending to actually skip it. Do you feel we should add this since it is also covered later in the docs?

Reply:
I think quick introductions are always good, to have a fast view of whats in there. For example:

*In section one we cover this and in section two this this and this*

Mainly to give the user the power to choose were to go, being informed.

Reply:
In this case i feel like the introduction is not that needed since the segments are relatively short with a descriptive enough title, also I would avoid putting this much information at the start of the documentation since the user doesn't have the tools (architectural knowledge contained in the rest of the doc) to decompress them.